content/recipes/authoring/kata-snippets/submission-tests.md

```diff
@@ -0,0 +1,88 @@
+---
+kind: recipe
+sidebar: authoring
+prev: /recipes/authoring/kata-snippets/sample-tests/
+next: /recipes/authoring/kata-snippets/preloaded/
+---
+
+# Writing Submission Tests
+
+This article contains a set of guidelines for kata authors or translators to create good test suites for their kata. They were collected to help ensure that kata, translations and their test suites are of sufficient quality and users' experience with tests will be as good as possible.
+
+The guidelines should be used by kata authors, translators and reviewers to verify whether new content about to be introduced to Codewars is of sufficient quality. Conformance with them should be verified before a kata or translation is published by its author and approved by a reviewer.
```

Comment:
there is sort of a "doubling" with these two sentences. I'd remove one, or merge them.

```suggestion
These guidelines should be used by kata authors, translators and reviewers to verify whether new contents about to be introduced to Codewars are qualitative enough before they are published or approved.
```

Reply:
Fixed. In this case, "content" is uncountable (like water).